HW9/controllers/product.controller.ts

```diff
@@ -15,6 +16,7 @@ ProductController.get(
   async (req, res) => {
     const product = await ProductService.getProduct(req.params.productId);
     if (!product) {
+      logger.error("No product with such id");
```

Comment:
This is out of scope for the task, but it's a good practice to log what data user sends to your application (unless, this is something sensitive).